### ISSUE TITLE
setPage not working

### ISSUE THREAD

**Issue by SabrinaMadX** [spatie/pdf-to-image#35]
Hi, i think your setPage isn't working.

i did something like this:

```php
for($i = 1; $i<($numpages+1); $i++){
$pdf->setPage($i)->saveImage($path.'/testcvrt_'.$i.'.png');
}
```

but all it ever converts is the first page of the pdf file. Same goes for 
```php
saveAllPagesAsImages()
```
It only converts the first page of the pdf file.

**Comment by freekmurze** [comment]
Have you tried this with a few different pdf's or does the problem only occur with a specific single pdf?

**Comment by SabrinaMadX** [comment]
i tried with the following files:
they're sample files i grabbed off the internet. 
[5pages_2.pdf](https://github.com/spatie/pdf-to-image/files/953968/5pages_2.pdf)
[multi.pdf](https://github.com/spatie/pdf-to-image/files/953969/multi.pdf)

**Comment by freekmurze** [comment]
This is fixed in `v1.3.3` of the package. Run `composer update` to pull it in.
